README.md

```diff
@@ -271,8 +271,17 @@ auth_opt_cache_refresh true
 
 auth_opt_auth_cache_seconds 30
 auth_opt_acl_cache_seconds 30
+auth_opt_auth_jitter_seconds 3
+auth_opt_acl_jitter_seconds 3
 ```
 
+The `auth_jitter_seconds` and `acl_jitter_seconds` allow to randomize the expiration used. The value used
+is cache_seconds +/- jitter_seconds. With above values (30 seconds for cache and 3 seconds for jitter), the
+expiration will be between 27 seconds and 33 seconds. The jitter is useful to reduce loopups storm that could
+occur every auth/acl_cache_seconds if lots of clients connected at the same time (for example
+after a server restart, all your clients may reconnect immediatly creating lots of entry expiring at the same time, unless
+a jitter is used). You can set jitter to 0 to disable this feature.
+
```

Comment:
```suggestion
`auth_jitter_seconds` and `acl_jitter_seconds` options allow to randomize cache expiration time by a given offset
The value used for expiring a cache record would then be `cache_seconds` +/- `jitter_seconds`. With above values (30 seconds for cache and 3 seconds for jitter), effective expiration would yield any value between 27 and 33 seconds. 
Setting a `jitter` value is useful to reduce lookups storms that could occur every `auth/acl_cache_seconds` if lots of clients connected at the same time, e.g. after a server restart when all clients may reconnect immediately creating lots of entries expiring at the same time.
You may omit or set jitter options to 0 to disable this feature.

```
Besides the suggestion, this would only be true when refreshing the cache on startup, as a non refreshed cache wouldn't suffer from a horde of clients reconnecting, right? I think trusting the user to not set `jitter` when having a persistent cache is good enough, but do you think we should have some `no_jitter` default in that case?

Reply:
This PR try to fix the following use case:
* You had hundred of clients connected
* You restart your mosquitto (like for upgrade).
* You don't have persistent cache
* All client got disconnected. All client reconnect (immediately in my case - at least within the few next seconds)
* All clients will trigger auth & acl checks at the same time. Since no persistent cache, it will storms the auth/acl backend. We can't avoid this one (without persistent cache).
* But without jitter, the cache expiration of all entry will be at the same time. Causing another storming of the backend on entries expiration (at least if client active enough, in my case the sent every 10 seconds).
* And this will continue every acl_cache_seconds

So:
* Yes this happend only if Mosquitto is restarted and mosquitto-go-auth don't have persistent cache
* (actually it happend also if for some reason lots of client connect at the same time - and stay connected)
* It don't try to fix the first storming, only subsequent
* With persistent cache, this jitter is likely not useful.

I'm not sure about the "`no_jitter` default" ? Do you just want that default value of jitter are 0 ? (which means no effect)

Reply:
What I meant was if e.g. the cache is persistent and not set to flush on restart, then we would default to no jitter even if the users provided one. But after some thought it may not be such a good idea.